addons/l10n_latam_check/models/account_payment.py

```diff
@@ -245,11 +256,24 @@ def _prepare_move_line_default_vals(self, write_off_line_vals=None, force_balanc
         # we dont check the payment method code because when deposited on bank/cash journals pay method is manual but we still change the label
         # we dont want this names on the own checks because it doesn't add value, already each split/check line will have it name
         elif (self.l10n_latam_new_check_ids or self.l10n_latam_move_check_ids) and self.payment_method_code != 'own_checks':
+            check_name = [check_name for check_name in (self.l10n_latam_new_check_ids | self.l10n_latam_move_check_ids).mapped('name') if check_name]
             document_name = (
                 _('Checks %s received') if self.payment_type == 'inbound' else _('Checks %s delivered')) % (
-                ', '.join((self.l10n_latam_new_check_ids or self.l10n_latam_move_check_ids).mapped('name'))
+                ', '.join(check_name)
             )
             res[0].update({
                 'name': document_name + ' - ' + ''.join([item[1] for item in self._get_aml_default_display_name_list()]),
             })
         return res
+
+    @api.depends('l10n_latam_move_check_ids')
+    def _compute_destination_account_id(self):
+        # EXTENDS 'account'
+        super()._compute_destination_account_id()
+        for payment in self:
+            if payment.l10n_latam_move_check_ids and (not payment.partner_id or payment.partner_id == payment.company_id.partner_id):
+                payment.destination_account_id = self.env.company.transfer_account_id.id
```

Comment:
```suggestion
                payment.destination_account_id = payment.company_id.transfer_account_id.id 
```
or even the parent with the branches?

Reply:
Maybe we can put that for the moment and correct later if needed, i will try and push the suggestion, thx for the review :smile:

Reply:
@jco-odoo Fixed thanks for the review :smile: